CHANGELOG.md

```diff
@@ -1,5 +1,63 @@
 For 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
 
+2.1.0 (February 4, 2020)
+========================
+
+New Features
+------------
+
+- A RemoteParticipant will now emit a "reconnecting" event when it is trying to
+  re-establish its signaling connection to the Room after a network disruption/handoff.
+  Once it has successfully reconnected to the Room, it will emit a "reconnected"
+  event. (JSDK-2662)
```

Comment:
Do you think we should explicitly mention 15-sec timeout?

Reply:
@syerrapragada Sure, that is the maximum time of detection, although it can happen sooner.